packages/maskbook/src/plugins/BarnBridge/apis/index.ts

```diff
@@ -86,15 +105,19 @@ function ConstructRedirectUrl(protocolName: string, coin: string) {
 }
 
 function PrettifyLiquidity(liquidity: string) {
```

Comment:
why revert the function?

Reply:
Ah i originally thought it was causing truncation issues but it wasn't the root cause of the accuracy problems - i meant to restore this back to the log algo. Let me push real quick